network.loki.Session.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id": "network.loki.Session",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "session",
+    "separate-locales": false,
+    "rename-desktop-file": "session-messenger-desktop.desktop",
+    "rename-icon": "session-messenger-desktop",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--own-name=org.kde.StatusNotifierItem-2-1"
```

Comment:
This won't really work in practice, you need `--own-name=org.kde.*` if you want a working tray icon

Reply:
Should be fixed as of 18a8112a3fbc06442357f0c8a58f0d41db0733f9